test/assert_relationship_test.exs

```diff
@@ -99,4 +99,9 @@ defmodule AssertRelationshipTest do
         assert "record with `id` 1 and `type` \"post\" was found but had mis-matching attributes" == error.message
     end
   end
+
+  test "will return the original payload" do
+    payload = assert_relationship(data(:payload), data(:author), as: "author", for: data(:post))
```

Comment:
Ah, right, which would be a severe limitations I ignore me 

Sent from my iPhone

> On Jul 26, 2016, at 11:51 AM, Brian Cardarella <notifications@github.com> wrote:
> 
> In test/assert_relationship_test.exs:
> 
> > @@ -99,4 +99,9 @@ defmodule AssertRelationshipTest do
> >          assert "record with `id` 1 and `type` \"post\" was found but had mis-matching attributes" == error.message
> >      end
> >    end
> > +
> > +  test "will return the original payload" do
> > +    payload = assert_relationship(data(:payload), data(:author), as: "author", for: data(:post))
> But we can't compose off that, right?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>